ros2pkg/ros2pkg/api/__init__.py

```diff
@@ -37,6 +38,13 @@ def get_prefix_path(package_name):
     return prefix_path
 
 
+def get_share_directory(package_name):
+    try:
+        return get_package_share_directory(package_name)
+    except PackageNotFoundError:
+        return None
```

Comment:
Since this function did nothing more than calling another function I would rather not add this to the API module.

Reply:
Removed.